vms/registration/templates/registration/acc_active_email.html

```diff
@@ -0,0 +1,5 @@
+{% autoescape off %}
+Hi {{ user.username }},
+Please click on the link to confirm your registration,
+http://{{ domain }}/registration/signup_volunteer/activate/{{ uid }}/{{token}}/
```

Comment:
`https://` can you share one sample `uid`? 🤔

Reply:
http://127.0.0.1:8000/registration/signup_volunteer/activate/MTA/4sx-fa50c211861e74e8def5/
Here MTA is the uid for username anjali